R/pkg/R/mllib.R

```diff
@@ -665,6 +665,7 @@ setMethod("predict", signature(object = "KMeansModel"),
 #' @param tol convergence tolerance of iterations.
 #' @param stepSize stepSize parameter.
 #' @param seed seed parameter for weights initialization.
+#' @param initialWeights initialWeights parameter for weights initialization.
```

Comment:
could you clarify that this should be a vector - also is there any restrictions on the length of this?